src/System.Data.SqlClient/ref/System.Data.SqlClient.Manual.cs

```diff
@@ -17,4 +17,9 @@ public sealed partial class SqlConnection : System.Data.Common.DbConnection
         // SqlConection expects IDisposable methods to be implemented via System.ComponentModel.Component, which it no longer inherits from
         override protected void Dispose(bool disposing) { }
     }
+    [System.ComponentModel.TypeConverter(typeof(SqlParameterConverter))]
```

Comment:
Why put this in the manual file?

Reply:
Because I needed to manually list the internal type.